### DISCUSSION TITLE
Is it just my configuration or users just loose their credentials out of nowhere?

### DISCUSSION THREAD

**Discussion by ruilvo** [magicbug/Cloudlog#1768] [Q&A]
Hello!

I wanted to deploy Cloudlog, and managed to do so with a docker instance. I host the code for the (very easy!) docker deployment [here](https://github.com/ruilvo/cloudlog-docker-compose). 

But I have this bug... My administrator account always work, but sometimes the user accounts loose their ability to login out of the blue until I change their password via the admin account. 

Is it just me, or is this happening to someone else?

Using the "master" branch.

**Comment by magicbug** [comment]
That sounds very strange

**Reply by ruilvo** [reply]
Yeah, like, the accounts work for a couple days/hours, and then they can only login after a password reset. I can't figure the problem out...

**Reply by magicbug** [reply]
Is it happening to multiple accounts? 

I'd enable logging in /applicaton/config/config.php and see if anything shows in the logs after a while but I cant think of anything that would cause it.

**Reply by ruilvo** [reply]
I'll enable logging and then check when it happens again, thanks.
